site/en/docs/handbook/components/index.md

```diff
@@ -663,6 +663,38 @@ Second Term
 : This is one definition of the second term.
 : This is another definition of the second term.
 
+## Tabs
+
+Use `web-tabs` component to display information in a form of horizontal tabs.
+Attribute `title` becomes the title of the corresponding tab panel.
+
+```html
+<web-tabs>
+  <web-tab title="Tab 1">
+    <p>I'm content of Tab 1</p>
```

Comment:
Could you show an example that adds a code snippet? I think if you add a newline character it should be possible. Something like this:

````html
<web-tab title="Tab 1">

```js
const hello = 'world';
```

</web-tab>
````